File: DataSummitSolution/DataSummitUI/src/app/drawings/models/drawingupload.ts
export class DrawingUpload {

    ProjectId: number
    TemplateId: number
    UserId: number
    File: string
    FileName: string
    FileType: string

    constructor(projectId?: number, templateId?: number, userId?: number,
                file?: string, fileName?: string, fileType?: string)
    {
        this.ProjectId = projectId || 0;
        this.TemplateId = templateId || 0;
        this.UserId = userId || 0;
        this.File = file || "";
        this.FileName = fileName || "";
        this.FileType = fileType || "";
    }
}

File: DataSummitSolution/DataSummitWeb/DTO/DrawingUpload.cs
namespace DataSummitWeb.DTO
{
    public class DrawingUpload
    {
        public int ProjectId { get; set; }
        public int TemplateId { get; set; }
        public string UserId { get; set; }
        public string FileName { get; set; }
        public string FileType { get; set; }
        public byte[] File { get; set; }

        public DrawingUpload()
        { }
    }
}
